### PR TITLE
[FIX 11.0] sale: typo in version and security group renamed

### PR DESCRIPTION
**Pull request by njeudy**
Sale module:

- Rename account.group_proforma_invoices security group
- Fix typo

Needed by #1196 

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/sale/migrations/11.0.1.1/pre-migration.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 - Nicolas JEUDY
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade
+
+_xmlid_renames = [
+('account.group_proforma_invoices', 'sale.group_proforma_sales'),
```
Comment (pedrobaeza) [inline addons/sale/migrations/11.0.1.1/pre-migration.py]: Please indent this line.

Comment (njeudy) [reply addons/sale/migrations/11.0.1.1/pre-migration.py]: @pedrobaeza : 👍 I amend my commit :)

**Comment by pedrobaeza** [comment]
Pylint error:
```
./addons/sale/migrations/11.0.1.1/pre-migration.py:11:1: E302 expected 2 blank lines, found 1
```

**Comment by njeudy** [comment]
@pedrobaeza thanks :) don't know why my linter doesn't work today ...

**Comment by njeudy** [comment]
@pedrobaeza need I do something for coveralls ?

**Pull request merged**